packages/frontend/editor-ui/src/components/Folders/MoveToFolderDropdown.vue

```diff
@@ -108,6 +108,10 @@ const onFolderSelected = (folderId: string) => {
 		type: selectedFolder.resource,
 	});
 };
+
+onMounted(() => {
+	void setTimeout(() => moveFolderDropdown.value?.focusOnInput());
```

Comment:
I would love to get rid of `void setTimeout` but all of the other approaches I tried here either don't work or are unnecessarily complex, so happy to keep this as is